packages/governance-app/src/Council/CouncilMembers.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { AccountId, BlockNumber, Tuple, Vector } from '@polkadot/types';
+import { AppContext } from '@substrate/ui-common';
+import { AddressSummary, Card, FadedText, Header, StackedHorizontal } from '@substrate/ui-components';
+import React, { useContext, useEffect, useState } from 'react';
+import { Observable } from 'rxjs';
+
+export function CouncilMembers () {
+  const { api, keyring } = useContext(AppContext);
+  const [activeCouncil, setActiveCouncil] = useState();
```

Comment:
I think here `useState([])` could make sense, so that you don't need the 2 `activeCouncil && ...` below

Reply:
fwiw it expects a Tuple which doesn't have a `.toArray()` method, so initializing in state actually is less legible- something like `useState(new Tuple({}, null))`

Reply:
Ah ok I see, then leave it () then